packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -640,7 +640,7 @@ class PubspecYaml {
         // place to insert our transitive dependencies.
         if (section == Section.dependencies)
           endOfDirectDependencies = output.length;
-          endOfDevDependencies = output.length;
+        endOfDevDependencies = output.length;
```

Comment:
This one looked strange. Not sure if this line should be inside or outside of the if section.

Reply:
I suspect this line should just be removed. It looks like a copy paste error and is almost certainly wrong.

Reply:
/cc @jonahwilliams blame suggests you made this change.  Can you confirm this line should be removed?

Reply:
If you look a couple lines down it gets set if you're in the expected section.  It looks like if it does work it's probably got bugs (counts the dependencies for the wrong section(s)).

Reply:
Line removed and tests still work(locally)